_sections/COVID-19/en/06-keep-the-main-thing-the-main-thing.md

```diff
@@ -1,10 +1,6 @@
 ## Keep the main thing the main thing
 [**\#FlattenTheCurve**](https://twitter.com/hashtag/FlattenTheCurve?src=hashtag_click)
 
-A lot of ink has been spilled on speculation about the "true" rate; however we are still in the early phase of the pandemic and it could
-take years to precisely determine the rate. BUT we know that it is somewhere **between .5% and 4 percent mortality; this range is more than
-adequate evidence to warrant decisive, immediate, large-scale preventive action**. Far and away, the most important thing to do is [flatten the
-curve of the epidemic](https://www.economist.com/briefing/2020/02/29/covid-19-is-now-in-50-countries-and-things-will-get-worse)
-so that our health systems can cope and to give time for the scientists to research vaccines and treatments.
+A lot of ink has been spilled on speculation about the "true" rate; however we are still in the early phase of the pandemic and it could take years to precisely determine the rate. Evidence to date is more than adequate evidence to warrant continued vigilance. Far and away, the most important thing to do is to find sustainable interventions that [flatten the curve of the epidemic](https://www.economist.com/briefing/2020/02/29/covid-19-is-now-in-50-countries-and-things-will-get-worse) so that our health systems can cope and to give time for the scientists to research vaccines and treatments, and to ensure that the pandemic proceeds slowly so that we can avoid surpassing the number of infections required to get past the peak (see explanation [here](https://twitter.com/CT_Bergstrom/status/1259219583325499393)).
```

Comment:
_Evidence to date is more than adequate evidence to warrant continued vigilance._ - please rephrase.

The second sentence is rather long and complex and might benefit from breaking up. 

Especially the last bit is hard to comprehend: _and to ensure that the pandemic proceeds slowly so that we can avoid surpassing the number of infections required to get past the peak_